packages/story-editor/src/components/header/buttons/test/publish.js

```diff
@@ -127,6 +133,14 @@ describe('PublishButton', () => {
     fireEvent.click(publishButton);
 
     expect(showPriorityIssues).toHaveBeenCalledTimes(1);
+
+    const publishButtons = screen.queryAllByRole('button', {
+      name: 'Publish',
+    });
+    // story details modal is open, now there is a second publish button.
+    expect(publishButtons).toHaveLength(2);
+    fireEvent.click(publishButtons[1]);
```

Comment:
comment here is super helpful! I wonder if there's a way to target the modal button. Maybe something like
```js
const publishButton = within(modal).getByRole('button', { name: /Publish/ });
```

that way we don't have to worry about button order?

Reply:
oh yeah duh, yes thanks!